data.sql

```diff
@@ -1,13 +1,136 @@
 /* Populate database with sample data. */
-
-INSERT INTO animals (name, date_of_birth,weight_kg, neutered, escape_attempts) 
-VALUES('Agumon','2020-02-03', 10.23, TRUE, 0);
-
-INSERT INTO animals (name, date_of_birth,weight_kg, neutered, escape_attempts) 
-VALUES('Gabumon','2018-11-15', 8, TRUE, 2);
-
-INSERT INTO animals (name, date_of_birth,weight_kg, neutered, escape_attempts) 
-VALUES('Pikachu','2021-01-07', 15.04, FALSE, 1);
-
-INSERT INTO animals (name, date_of_birth,weight_kg, neutered, escape_attempts) 
-VALUES('Devimon','2017-05-12', 11, FALSE, 5);
\ No newline at end of file
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Agumon', '2020-02-03', 10.23, TRUE, 0);
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Gabumon', '2018-11-15', 8, TRUE, 2);
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Pikachu', '2021-01-07', 15.04, FALSE, 1);
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Devimon', '2017-05-12', 11, FALSE, 5);
+------------------------ Add new data -------------------------
+BEGIN;
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Charmander', '2020-02-08', 11, FALSE, 0);
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Plantmon', '2021-11-15', 5.7, TRUE, 2);
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Squirtle', '1993-04-02', 12.13, FALSE, 3);
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Angemon', '2005-06-12', 45, TRUE, 1);
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Boarmon', '2005-06-07', 20.4, TRUE, 7);
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Blossom', '1998-10-13', 17, TRUE, 3);
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+VALUES('Ditto', '2022-05-14', 22, TRUE, 4);
+COMMIT;
+-- update the animals table by setting the species column to unspecified
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Kindly verify that changes were made before and after ROLLBACK. Hint; `SELECT species from animals;`

```suggestion
BEGIN;
UPDATE animals
SET species = 'unspecified';

SELECT species from animals;`

ROLLBACK;

SELECT species from animals;`
```